src/content/docs/guides/node-setup/configure-tls.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Configure TLS
+description: Set up TLS encryption for operators and platform connections
+---
+
+Tenzir supports Transport Layer Security (TLS) for encrypting network
+connections. You can configure TLS settings centrally in `tenzir.yaml` so they
+apply to all compatible operators, or override them per-operator as needed.
+
+:::tip[Platform connection]
+To configure TLS for the connection to the Tenzir Platform, see
+[Configure TLS for the Platform connection](/guides/node-setup/configure-a-node#configure-tls-for-the-platform-connection).
+:::
+
+## Node-level TLS configuration
+
+Instead of passing TLS options to each operator individually, configure them
+once under `tenzir.operator-tls` in your `tenzir.yaml`:
+
+```yaml title="tenzir.yaml"
+tenzir:
+  operator-tls:
+    tls-min-version: "1.2"
+    tls-ciphers: "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256"
+    cacert: "/etc/ssl/certs/ca-certificates.crt"
+```
+
+These settings apply automatically to operators like `load_http`, `save_http`,
+`load_tcp`, `save_tcp`, `to_opensearch`, `from_opensearch`, `to_splunk`,
+`save_email`, and `to_fluent_bit`.
+
+### Available options
+
+| Option                   | Description                                                         |
+| ------------------------ | ------------------------------------------------------------------- |
+| `enable`                 | Enable TLS on all operators that support it                         |
+| `skip-peer-verification` | Disable certificate verification (not recommended for production)   |
+| `cacert`                 | Path to a CA certificate bundle for server verification             |
+| `certfile`               | Path to a client certificate file                                   |
+| `keyfile`                | Path to a client private key file                                   |
+| `tls-min-version`        | Minimum TLS protocol version: `"1.0"`, `"1.1"`, `"1.2"`, or `"1.3"` |
+| `tls-ciphers`            | OpenSSL cipher list string                                          |
+
+### Precedence
+
+Arguments passed directly to operators always take precedence over node-level
+settings. This lets you override specific connections when needed:
+
+```tql
+// Uses node-level TLS settings
+load_http "https://api.example.com/data"
+
+// Overrides with a custom certificate
+load_http "https://internal.example.com/data", cacert="/path/to/internal-ca.crt"
```

Comment:
More fundamental comment: have we thought of a _single_ `tls` record that we can pass, instead of a list of multiple operators?